src/Client.php

```diff
@@ -222,6 +249,7 @@ public function executeWorkspaceQuery(string $branchId, string $workspaceId, arr
             }
         }
 
+        /** @phpstan-ignore-next-line */
```

Comment:
co se tady ignoruje? vypadá to, že už je to všechno zkontrolovaný

Reply:
array{queryJobId: string, status: string, statements: array<array<string, mixed>>, results: array<array{columns: array<array{name: string, type: 'text'}>, data: array<array<int, string>>, status: string, rowsAffected: int}>} - should return
array{queryJobId: string, status: 'completed', statements: mixed, results: array<int<0, max>, array{columns: array<int, array{name: string, type: 'text'}>, data: array<array<int, string>>, status: string, rowsAffected: int}>} -  but returns

Reply:
nojo, tak to je ale fixnutelny, on to nebere, protože tady je to celý v ifu https://github.com/keboola/query-api-php-client/pull/5/files#diff-40d7473316ef0bb3ba4c4d937b16ae3d1cc47e0d65815034ee2b5be6a74cb11eR241 kterej nevyhodí invalid response